src/views/validated-designs/server.ts

```diff
@@ -115,7 +116,7 @@ export function getHvdCategoryGroups(): HvdCategoryGroup[] | null {
 				slug,
 				title: metadata.title,
 				description: metadata.description,
-				product,
+				product: product as Exclude<ProductSlug, 'sentinel'>,
```

Comment:
had to update this because `<IconTileLogo size="large" productSlug={category.product} />` was complaining about the type for `productSlug`

Reply:
hmmmm okay.